CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # myuw-search versions
 
+## 1.5.0
+
+* Added role='search' to the search form
+* Added label tag associated with search input. Accessible for assistive technologies users only
```

Comment:
```suggestion
```

I think this line about a `label` tag is a remnant from a prior version of this changeset and the changeset in its latest form does not add a `label` tag.